### PR TITLE
fix: sometimes the same token can return more than one user

### PR DESCRIPTION
**Pull request by jimsynz**
this happens when the same user is signed in in more than one browser.

### CODE REVIEW

**Comment by zachdaniel** [comment]
Same user logged in on different browsers, the ash hq code probably has this same problem. I'd suggest `Ash.Query.limit(query, 1)` instead.

**Comment by zachdaniel** [comment]
Then you can still use `read_one`

**Comment by zachdaniel** [comment]
Also the pattern match there doesn't account for no return value, which is acceptable from that function.

**Comment by juddey** [comment]
Thanks for the suggestion Zach! :)

File: lib/tictactoe.ex [hunk 1]
```diff
@@ -0,0 +1,3 @@
+defmodule TicTacToe do
+  use Playdate.GameProvider, name: "Tic tac toe", min_players: 1, max_players: 2
```
Comment (juddey) [inline lib/tictactoe.ex:2]:
```suggestion
@moduledoc """
  Demonstration Module to illustrate parameters passed in for a Game
"""
  use Playdate.GameProvider, name: "Tic tac toe", min_players: 1, max_players: 2
```

**Review by juddey — APPROVED** [review]

File: lib/tictactoe.ex (hunk 1 above)
Comment (jimsynz) [reply lib/tictactoe.ex]: accidentally committed. removed.

**Comment by jimsynz** [comment]
Thanks for the suggestions @zachdaniel  - fixed.

**Review by zachdaniel — APPROVED** [review]

**Pull request merged**